modules/machineset-yaml-osp-sr-iov-port-security.adoc

```diff
@@ -0,0 +1,125 @@
+* The compute security group
+* Attachment to the machines network and subnet
+
+[NOTE]
+====
+Only parameters that are specific to SR-IOV deployments where port security is disabled are described in this sample. To review a more general sample, see Sample YAML for a machine set custom resource that uses SR-IOV on {rh-openstack}".
+====
+
+.An example machine set that uses SR-IOV networks and has port security disabled
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_ID>
+    machine.openshift.io/cluster-api-machine-role: <node_role>
+    machine.openshift.io/cluster-api-machine-type: <node_role>
+  name: <infrastructure_ID>-<node_role>
+  namespace: openshift-machine-api
+spec:
+  replicas: <number_of_replicas>
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_ID>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_ID>
+        machine.openshift.io/cluster-api-machine-role: <node_role>
+        machine.openshift.io/cluster-api-machine-type: <node_role>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role>
+    spec:
+      metadata: {}
+      providerSpec:
+        value:
+          apiVersion: openstackproviderconfig.openshift.io/v1alpha1
+          cloudName: openstack
+          cloudsSecret:
+            name: openstack-cloud-credentials
+            namespace: openshift-machine-api
+          flavor: <nova_flavor>
+          image: <glance_image_name_or_location>
+          kind: OpenstackProviderSpec
+          ports:
+            - allowedAddressPairs: <1> 
+              - ipAddress: <API_VIP_port_IP>
+              - ipAddress: <ingress_VIP_port_IP>
+              fixedIPs:
+                - subnetID: <machines_subnet_UUID> <2>
+              nameSuffix: nodes
+              networkID: <machines_network_UUID> <2>
+              securityGroups:
+                  - <compute_security_group_UUID> <3>
+            - networkID: <SRIOV_network_UUID>
+              nameSuffix: sriov
+              fixedIPs:
+                - subnetID: <SRIOV_subnet_UUID>
+              tags:
+                - sriov
+              vnicType: direct
+              portSecurity: False
+          primarySubnet: <machines_subnet_UUID>
+          serverMetadata:
+            Name: <infrastructure_ID>-<node_role>
+            openshiftClusterID: <infrastructure_ID>
+          tags:
+          - openshiftClusterID=<infrastructure_ID>
+          trunk: false
+          userDataSecret:
+            name: worker-user-data
+          configDrive: True
+----
+<1> Define allowed address pairs for the API and ingress ports here.
+<2> Specify the machines network and subnet here.
+<3> Specify the compute machines security group here.
```

Comment:
two nits:
* Why not start <1> with "Specify the allowed..."?
* Is there a benefit to using "here" rather than omitting it?